vb-be/src/modules/room/room.constants.ts

```diff
@@ -9,10 +10,13 @@ export interface ICreateRoom {
   maxUsers?: number;
 }
 
-export interface ISocketRedisRoom {
+export interface IRedisRoom {
   _id: Types.ObjectId;
   name: string;
   description?: string;
   tags?: string[];
-  host: string;
+  host: {
+    _id: Types.ObjectId;
+    username: string;
+  };
```

Comment:
I figured it may be beneficial to keep username in redis, so we can easily display the username of the user in the room card, if we'd like

Reply:
shouldn't this be just the host? like we had discussed? and then have the `populate` function fill it up?